lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -8,20 +8,20 @@ edition = "2018"
 crate-type = ["staticlib"]
 
 [dependencies]
-bitflags = "2.3.2"
+bitflags = "2.3.3"
```

Comment:
Same remarks as https://github.com/gravitational/teleport/pull/28584#discussion_r1250973671 - operator changes look breaking to me, but I'm unsure if it makes a difference here.